.circleci/config.yml

```diff
@@ -158,18 +169,39 @@ jobs:
           name: Install psql
           command: |
             sudo apt-get update -y && sudo apt-get install -y postgresql-client
+      - run:
+          name: Install mysql
+          command: |
+            sudo apt-get update -y && sudo apt-get install -y mysql-client
       - run:
           name: Wait for vertica
           command: |
             # Bail out trying after 30 seconds
             end=$((SECONDS+30))
             echo "Starting at second ${SECONDS:?} - ending at ${end:?}"
+            # TODO: Make into alias
             while ! vsql -h 127.0.0.1 -U dbadmin -c 'select 1;' && [[ "${SECONDS:?}" -lt "${end:?}" ]]
             do
               echo "Waiting for vertica..."
               sleep 5
             done
             vsql -h 127.0.0.1 -U dbadmin -c 'select 1;'
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker